src/ui/public/timepicker/timepicker.js

```diff
@@ -24,7 +24,9 @@ module.directive('kbnTimepicker', function (quickRanges, timeUnits, refreshInter
       to: '=',
       mode: '=',
       interval: '=',
-      activeTab: '='
+      activeTab: '=',
+      onFilterSelect: '=',
```

Comment:
The "Angular way" of doing this would be to refer to the callbacks as `onFilterSelect: '&'` and then in the markup pass in the callback like this: `on-filter-select="updateFilter()"`.